designs/index.md

```diff
@@ -53,7 +63,6 @@ or [using Markdown](#using-markdown). Read below about for a
 Proposals that have a user-visible impact must have a section documenting the
 impact on backward compatibility (and a rollout plan if needed).
 
-
 ### Create a Pull Request
 
 The author creates a Pull Request (PR) to add the document to
```

Comment:
To share your design doc, create a pull request (PR) to add the document to the [the design index](https://github.com/bazelbuild/proposals). Add your markdown file or a document link to your PR.

When possible,  [choose a lead reviewer](#how-to-choose-a-lead-reviewer) and cc other reviewers. If you don't choose a lead reviewer, a Bazel maintainer will assign one to your PR.

After you send your PR, the reviewers can make preliminary comments during the
code review. For example, the lead reviewer can suggest extra reviewers, or
point out missing information. The lead reviewer approves the PR when they
believe the review process can start. It doesn't mean the proposal is perfect or
will be approved; it means that the proposal contains enough information to
start the discussion.

Reply:
Take a look at the following paragraphs and update them to be direct address where possible, please.

Reply:
Done.